### PR TITLE
Update to 0.9.10, and update dependencies

### PR DESCRIPTION
**Pull request by corecontingency**
Solves issue #4 . I also made a beta branch with unstable version 0.9.11, but I cannot seem to open a pull request to merge it.

### CODE REVIEW

**Comment by corecontingency** [comment]
Fails in the `Check for right id in AppStream xml` step with:
```
zgrep "<id>io.github.EndlessSky.endless-sky\(\.\w\+\)*\(.desktop\)\?</id>" builddir/*/share/app-info/xmls/io.github.EndlessSky.endless-sky.xml.gz
 in dir /srv/buildbot/worker/build-x86_64-4/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'zgrep "<id>io.github.EndlessSky.endless-sky\\(\\.\\w\\+\\)*\\(.desktop\\)\\?</id>" builddir/*/share/app-info/xmls/io.github.EndlessSky.endless-sky.xml.gz'
 using PTY: False
program finished with exit code 1
elapsedTime=0.017015
```

Confused why this is failing, as it builds fine on my computer.
```
flatpak run org.freedesktop.appstream-glib validate io.github.EndlessSky.endless-sky.appdata.xml
io.github.EndlessSky.endless-sky.appdata.xml: OK`
```
Even running the above on the appdata file in the build directory says it is fine.

Any ideas?

File: endless-sky.appdata.xml [hunk 1]
```diff
@@ -1,10 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
-  <id>io.github.EndlessSky.endless-sky</id>
-  <launchable type="desktop-id">endless-sky.desktop</launchable>
+  <id>io.github.endless_sky.endless_sky</id>
```
Comment (nedrichards) [inline endless-sky.appdata.xml]: the capitalization is wrong here. this is causing the appdata failure.

**Comment by nedrichards** [comment]
@corecontingency I also pushed an unchanged 'beta' branch which you should be able to send a PR against now.

**Comment by corecontingency** [comment]
If it builds fine I will send a PR for the beta branch. Thanks!

**Comment by corecontingency** [comment]
@nedrichards Ready to merge both PRs. I'd be happy to maintain this flatpak if you need someone!

**Comment by corecontingency** [comment]
@barthalion Hey, can you merge this PR and PR #6 ? Thank you.

**Comment by corecontingency** [comment]
@barthalion Thanks!

**Pull request merged**